src/css/layout/ScrollbarLayout.css

```diff
@@ -27,23 +27,18 @@
 .ScrollbarLayout/mainHorizontal {
   height: var(--scrollbar-size);
   left: 0;
-  transition-property: background-color height;
```

Comment:
Moved to the thumb element (see `:after`) which is the actual element that needs the animation